### DISCUSSION TITLE
Anyone recommend a good forum

### DISCUSSION THREAD

**Discussion by andy5995** [fffaraz/awesome-cpp#1253] [General]
I searched the README for "forum" and no results were found. "Discussion" turned up one result.

Can anyone recommend a few good cpp forums?

Yes, I know I can search Duck Duck Go or some other engine, but I'm just looking for recommendations from actual people.

**Comment by MartinDelille** [comment]
https://cpplang.slack.com/ ?

**Comment by andy5995** [comment]
Ah, thanks. I wasn't very clear though. I meant a web forum not a chat forum.

**Comment by yetanothergeek** [comment]
https://www.reddit.com/r/Cplusplus/

**Comment by andy5995** [comment]
I should have started this as a discussion instead of an issue. Admin wanna convert this issue to a discussion?

**Reply by Forehm** [reply]
4chan
